Tools/TestRunnerShared/spi/UIKitSPIForTesting.h

```diff
@@ -100,22 +101,6 @@ extern NSNotificationName const _UIWindowSceneEnhancedWindowingModeChanged;
 
 WTF_EXTERN_C_END
 
-#if HAVE(UIFINDINTERACTION)
-
-@interface _UIFindInteraction : NSObject <UIInteraction>
-@end
-
-@interface _UIFindInteraction (Staging_84486967)
-
-- (void)presentFindNavigatorShowingReplace:(BOOL)replaceVisible;
-
-- (void)findNext;
-- (void)findPrevious;
-
-@end
-
-#endif // HAVE(UIFINDINTERACTION)
```

Comment:
I think we'll still need this to support open source iOS builds

Reply:
We can certainly not call the interface `Staging` anymore though! but we will need to maintain this in `#else // USE(APPLE_INTERNAL_SDK)`

```objc
#if HAVE(UIFINDINTERACTION)
@interface _UIFindInteraction : NSObject <UIInteraction>
- (void)presentFindNavigatorShowingReplace:(BOOL)replaceVisible;
- (void)findNext;
- (void)findPrevious;
@end
#endif
```

Reply:
Actually, it looks like we only use `presentFindNavigatorShowingReplace` in our test code, so this should suffice:

```
```objc
#if HAVE(UIFINDINTERACTION)
@interface _UIFindInteraction : NSObject <UIInteraction>
- (void)presentFindNavigatorShowingReplace:(BOOL)replaceVisible;
@end
#endif
```
```